test/Faker/Provider/ne_NP/PaymentTest.php

```diff
@@ -11,37 +11,68 @@ public function testCommercialBank(): void
     {
         $str = $this->faker->commercialBank();
         self::assertIsString($str);
+
+        $this->faker->seed(1);
+
+        $strNext = $this->faker->commercialBank();
+        self::assertSame($str, $strNext);
     }
 
     public function testDevelopmentBank(): void
     {
         $str = $this->faker->developmentBank();
         self::assertIsString($str);
+
+        $this->faker->seed(1);
```

Comment:
You do not have to seed Faker, it is done in the `TestCase` class.

Reply:
@bram-pkg I tested that without seeding. It's not working for sure.